README.md

```diff
@@ -12,13 +12,31 @@ For example to use in README's or as webpage static assets.
 
 ```sh
 # 1. Use the main sourcecred's score feature to export scores to a file.
-export SOURCECRED_GITHUB_TOKEN=YOUR_GITHUB_TOKEN
-node PATH_TO_SOURCECRED/bin/sourcecred.js load sourcecred/sourcecred
+SOURCECRED_GITHUB_TOKEN=YOUR_GITHUB_TOKEN \
+	node PATH_TO_SOURCECRED/bin/sourcecred.js load sourcecred/sourcecred
 node PATH_TO_SOURCECRED/bin/sourcecred.js scores sourcecred/sourcecred > scores.json
 
 # 2. Install this package's dependencies.
 yarn
 
 # 3. Generate a contributor wall SVG.
-./bin/contributor-wall-svg.js < scores.json > contributors.svg
+SOURCECRED_GITHUB_TOKEN=YOUR_GITHUB_TOKEN \
+	./bin/contributor-wall-svg.js < scores.json > contributors.svg
+```
+
+Or using the Docker equivalent of this.
+
+```sh
+# 1. Use the main sourcecred image to export scores to a file.
+SOURCECRED_GITHUB_TOKEN=YOUR_GITHUB_TOKEN \
+	docker run --rm -ti -v sourcecred_data:/data -e SOURCECRED_GITHUB_TOKEN sourcecred/sourcecred load sourcecred/sourcecred
```

Comment:
nit: These lines are long enough to be wrapping on my screen, which
makes the content ambiguous and hard to read; please hard-wrap at 80.

Reply:
Applied here and https://github.com/sourcecred/sourcecred/pull/1288